### DISCUSSION TITLE
CloudSpecs vs ExtendedSpecs

### DISCUSSION THREAD

**Discussion by whoDoneItAgain** [aws-cloudformation/cfn-lint#2701] [General]
I am trying to understand the difference between the CloudSpecs and ExtendedSpecs folders. Looking at #2698 CloudSpecs/us-east-1.json is showing the PerformanceInsightsRetentionPeriod AllowedValues I added in #2696. Does this mean that the automated process to pull in the AWS published CloudFormation specifications now included this? Are the ExtendedSpecs merged into the CloudSpec folder files? Any guidance on this and the general process how would be great so I can contribute more efficiently. 

Thanks

**Comment by kddejong** [comment]
This all changes in v1... but for v0 the extended specs are pulled when running cfn-lint --update-specs and then applied to the specs in CloudSpecs.

**Reply by PatMyron** [reply]
Described in more detail here: https://github.com/aws-cloudformation/cfn-lint/issues/903#issuecomment-595356853

**Comment by whoDoneItAgain** [comment]
ah. makes sense. i was hoping my contribution wasn't for not. It be one thing if it were weeks or months from now but literally the next day. That would have been depressing.
